backend/Dockerfile

```diff
@@ -15,4 +15,5 @@ RUN chmod +x /code/scripts/start.sh
 
 COPY . /code/
 
-ENTRYPOINT ["/code/scripts/start.sh"]
+# ENTRYPOINT ["/code/scripts/start.sh"]
+ENTRYPOINT exec uvicorn main:app --reload --host 0.0.0.0 --port 5050
```

Comment:
Hey, thanks for your contribution. Please can you remove this line :)

Reply:
ahh nevermind, main branch already delete the problem so I think this pull is no longer needed
https://github.com/StanGirard/quivr/commit/9f2cd7b7b67f25d94e2b562b34aebb0a5afbecc3#diff-76fa5b4102e91223400a4c3a2a7b332b8a53de873a40eee02bfa5486a62e6c8cL282